nailgun/entities.py

```diff
@@ -4005,9 +4006,12 @@ def update(self, fields=None):
 
     def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
-        return {
-            u'organization': super(Organization, self).update_payload(fields)
-        }
+        org_payload = super(Organization, self).update_payload(fields)
+        payload = {u'organization': org_payload}
+        if 'redhat_repository_url' in org_payload:
+            rh_repo_url = org_payload['redhat_repository_url']
+            payload['redhat_repository_url'] = rh_repo_url
```

Comment:
Extra variables just to fit the max line length or for other purpose?
I've tried to play with it a bit, result is a bit more readable for me, but that's arguable as it still doesn't look great:
```python
        payload = {
            u'organization': super(Organization, self).update_payload(fields),
        }
        if 'redhat_repository_url' in payload['organization']:
            payload['redhat_repository_url'] = payload['organization'].pop(
                'redhat_repository_url')
        return payload
```